parcels/field.py

```diff
@@ -310,8 +310,13 @@ def eval(self, time, x, y):
         return self.units.to_target(value, x, y)
 
     def ccode_eval(self, var, t, x, y):
-        return "temporal_interpolation_linear(%s, %s, %s, %s, %s, %s, &%s)" \
-            % (x, y, "particle->xi", "particle->yi", t, self.name, var)
+        # Casting interp_method to int as easier to pass on in C-code
+        if self.interp_method is 'linear':
+            interp_method = 0
```

Comment:
Hmm, ok. All I was after was that we pass in the string `NEAREST` into the C kernel rather than its equivalent `1` for readability of the kernel code. I've just pushed a small adjustment for this purpose.

That being said, encapsulating this into a custom utility class to avoid string comparison on the Python side is probably a good idea but since SciPy expects the string it seems a little pointless at this point.